app/screens/channel/channel_post_list/channel_post_list.js

```diff
@@ -116,12 +118,12 @@ export default class ChannelPostList extends PureComponent {
         }
     };
 
-    loadMorePosts = () => {
+    loadMorePosts = debounce(() => {
         if (this.props.loadMorePostsVisible) {
             const {actions, channelId} = this.props;
             actions.increasePostVisibility(channelId);
         }
-    };
+    }, 100);
```

Comment:
I'd go for 500 if we are going this way

Reply:
also we can check if it's not called X milliseconds after first component render/channel id change?
that's my best idea to workaround this old RN bug - as you know I've been working on hacking a fix recently

Reply:
@koxen why 500?

Reply:
@enahum no particular reason. I'd go with 300 or 500 because 100 seems like too little to stop it.